ReactAndroid/src/main/java/com/facebook/react/devsupport/DevSupportManager.java

```diff
@@ -367,10 +367,18 @@ public void onReactInstanceDestroyed(ReactContext reactContext) {
   }
 
   public String getSourceMapUrl() {
+    if (mJSAppBundleName == null) {
+      return "";
+    }
+
     return mDevServerHelper.getSourceMapUrl(Assertions.assertNotNull(mJSAppBundleName));
   }
 
   public String getSourceUrl() {
+    if (mJSAppBundleName == null) {
+      return "";
+    }
+
```

Comment:
Dev support should be disabled when mJSAppBundleName is not provided. Otherwise there is no way to "reload JS" or to fetch the source map